pygmt/clib/session.py

```diff
@@ -1449,9 +1449,11 @@ def virtualfile_from_data(
                 _data.extend(extra_arrays)
         elif kind == "matrix":  # turn 2D arrays into list of vectors
             try:
+                # pandas.Series will be handled below like a 1d numpy ndarray
+                assert not hasattr(data, "to_frame")
```

Comment:
```suggestion
                assert not isinstance(data, pd.Series)
```
I think this is more readable.

Reply:
Nevermind, I missed the note about dask support.